src/dfx/src/commands/nns/import.rs

```diff
@@ -61,14 +61,20 @@ pub async fn exec(env: &dyn Environment, opts: ImportOpts) -> DfxResult {
 /// The "local" entries at the remote URL are often misssing or do not match our NNS installation.
 /// Always set the local values per our local NNS deployment.  We have all the information locally.
 fn set_local_nns_canister_ids(logger: &Logger, config: &mut Config) -> DfxResult {
```

Comment:
Note: `NNS_CORE` and `NNS_FRONTEND` have different types so we cannot simply do `NNS_CORE.iter().chain(NNS_FRONTEND.iter())`